_layouts/download.html

```diff
@@ -49,6 +49,12 @@
 {% highlight groovy %}
 testCompile '{{ page.groupId }}:{{ page.artifactId }}:{{ page.version }}'
 {% endhighlight %}
+
+<h3>Use in SBT</h3>
+{% highlight scala %}
+libraryDependencies += "{{ page.groupId }}" %% "{{ page.artifactId }}" % "{{ page.version }}" % "test"
```

Comment:
I don't have experience with SBT but I think the `%%` should be replaced by `%` because System Rules is not compiled for multiple Scala versions.

Reply:
You're right!
I took forever to do it, but it's fixed.